test/planwise/component/scenarios_test.clj

```diff
@@ -121,11 +121,13 @@
           projects2   (:planwise.component/projects2 system)
           project     (projects2/get-project projects2 project-id)
           scenario-id (scenarios/create-initial-scenario store project)
-          scenario    (scenarios/get-scenario store scenario-id)]
+          scenario    (scenarios/get-scenario store scenario-id)
+          new-scenario (assoc scenario
+                              :changeset [{:action "create-site" :site-id "new.1" :investment 10000 :capacity 50 :location {:lat 0 :lon 0}}]
+                              :label "sub-optimal"
+                              :investment 10000)]
       (do
-        (scenarios/update-scenario store project scenario)
-        (let [updated-scenario (scenarios/get-scenario store scenario-id)]
-          (is (some? (:updated-at updated-scenario)))
-          (is (:changeset updated-scenario) [])
-          (is (:investment updated-scenario) (:investment scenario))
-          (is (:state scenario) (:state updated-scenario)))))))
+        (scenarios/update-scenario store project new-scenario)
```

Comment:
if assert is used to validate label is not initial, then here
`(is (thrown? java.lang.AssertionError (scenarios/update-scenario store project new-scenario)))`
is probably needed.

ref: http://clojure.github.io/clojure/clojure.test-api.html and https://clojuredocs.org/clojure.core/assert